src/bindings/python/__init__.py

```diff
@@ -49,7 +53,104 @@ class Object:
         setattr(base if new_base is None else new_base, enum, obj)
 
 
-convert(Generator, 'ModelType', [
+convert(AnalyserEquation, 'Type', [
+    'TRUE_CONSTANT',
+    'VARIABLE_BASED_CONSTANT',
+    'RATE',
+    'ALGEBRAIC',
+])
+convert(AnalyserEquationAst, 'Type', [
+    'ASSIGNMENT',
+
+    # Relational and logical operators.
```

Comment:
I know I've asked somewhere else ... but does this need the SEP too? or is that taken into account when we make a double?

Reply:
I imagine you are talking about `cn` elements that make use of the e-notation, e.g.
```xml
<cn cellml:units="dimensionless" type="e-notation">
    123
    <sep/>
    99
</cn>
```

If so, then we do indeed account for it when analysing a model. IOW, the analyser doesn't tack the fact that a number uses an e-notation. It just tracks the number. So, for the above example, it will track the fact that we have a number which value is `123e99` (see [here](https://github.com/agarny/libcellml/blob/7db0034777a296358d0fbd8d5fdbe98fd45aa3ca/src/analyser.cpp#L824-L831)).